README.md

```diff
@@ -1,41 +1,35 @@
-# Web Science
-A library of reusable functionality for [Mozilla Rally](https://support.mozilla.org/en-US/kb/about-firefox-pioneer) studies.
+# WebScience
+WebScience is an open-source library for building browser-based research studies, including on [Rally](https://rally.mozilla.org/).
 
-## Requirements
-* [Node.js](https://nodejs.org/en/), available via [Homebrew](https://brew.sh/) on macOS
-* [Mozilla web-ext](https://extensionworkshop.com/documentation/develop/getting-started-with-web-ext/)
+## Design Goals
+* Reduce the barriers to implementing browser-based research studies, by providing production-quality functionality that is commonly required, difficult to implement correctly, and difficult to maintain.
+* Standardize measurement methods across browser-based research studies.
+* Advance the state of the art in browser-based research methods, such as by providing a sophisticated model for user attention to web content.
+* Encourage researchers to practice data minimization when implementing studies.
+* Provide an API that is built on and consistent with [WebExtensions](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions), and that is compatible with all major web browsers.
 
-## Example Usage
-See the [News and Disinformation Study](https://github.com/citp/news-disinformation-study), which we recommend using as a template.
+## Contributors
+The primary contributors to WebScience are researchers at the [Princeton University Center for Information Technology Policy](https://citp.princeton.edu/) and the [Mozilla Rally](https://rally.mozilla.org/) team.
 
-## Architecture
-The Web Science library consists of two types of modules: measurement modules that collect a specific type of data, and utility modules that provide supporting functionality. A Firefox Pioneer study can optionally take advantage of the Web Science library (or a subset of the library) for faster development and review. We recommend including the repository for Web Science as a Git subtree in the repository for a Firefox Pioneer study.
+## Getting Started with WebScience
+The easiest way to get started is with the Rally [study template](https://github.com/mozilla-rally/study-template), which has already been configured to integrate the WebScience library.
```

Comment:
Hmm, I don't think the study template has been configured to use WebScience... the first Rally Study will.

Reply:
https://github.com/mozilla-rally/study-template/pull/94